### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**
This has the normal three test, build and deploy jobs the other lambdas have.

It also has the check rules job. This runs at 07:20 on a Monday and checks if the rules have changed and if they have, if they still work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Run with coverage, missing lines and junit output `python -m pytest --cov-report
 
 ## Yara rules checks.
 
-There is a Jenkins job [TDR Antivirus Rule Update Check](https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Rule%20Update%20Check/) which is run on a schedule from the Jenkinsfile-check-rules file. This carries out the following steps. 
+There is a GitHub actions job [TDR Check Antivirus Rules](https://github.com/nationalarchives/tdr-antivirus/actions/workflows/check_rules.yml) which is run on a schedule from the .github/workflows/check_rules file. This carries out the following steps.
```
Comment (TomJKing) [inline README.md]: Maybe mention the time the schedule is run?

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment, feel free to ignore it.

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md]: Yep makes sense, I've added it in.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**